src/libmongoc/src/mongoc/mongoc-uri.c

```diff
@@ -1526,6 +1526,23 @@ mongoc_uri_finalize_directconnection (mongoc_uri_t *uri, bson_error_t *error)
    return true;
 }
 
+static void
+mongoc_uri_check_for_genuine_hosts (const char *hosts)
+{
+   char *hosts_lowercase = lowercase_str_new (hosts);
+
+   if (strstr (hosts_lowercase, ".cosmos.azure.com")) {
+      MONGOC_INFO ("Azure Cosmos DB is not genuine");
```

Comment:
```suggestion
      // CDRIVER-4682: MONGOC_INFO content is TBD.
```
Suggest refraining from emitting a message yet; instead, leave placeholders for now until DRIVERS-2583 has been finalized.

Reply:
I wasn't planning to merge this until we had final copy. Replacing `MONGOC_INFO` with a comment would render the tests irrelevant.